editor/src/hover/di.config.ts

```diff
@@ -16,11 +16,14 @@ import {
   TYPES
 } from 'sprotty';
 import { PopupPositionUpdater } from 'sprotty/lib/features/hover/popup-position-updater';
+import { IVY_TYPES } from '../types';
 import { IvyHoverMouseListener } from './hover';
 
 const ivyHoverModule = new ContainerModule((bind, _unbind, isBound) => {
+  bind(IvyHoverMouseListener).toSelf().inSingletonScope();
+  bind(TYPES.MouseListener).toService(IvyHoverMouseListener);
```

Comment:
why do you need to register both?

Reply:
Because `TYPES.MouseListener` is used multiple times and when I try to inject `IvyHoverMouseListener` the result is ambiguous. Thus, I use `IVY_TYPES.IvyHoverMouseListener` to inject.